support/langkit_support-lexical_envs_impl.adb

```diff
@@ -621,19 +667,42 @@ package body Langkit_Support.Lexical_Envs_Impl is
       Invalidate_Cache (Env);
    end Reference;
 
-   ---------
-   -- Get --
-   ---------
+   ------------------
+   -- Get_Internal --
+   ------------------
 
    procedure Get_Internal
-     (Self          : Lexical_Env;
-      Key           : Symbol_Type;
-      Lookup_Kind   : Lookup_Kind_Type := Recursive;
-      Rebindings    : Env_Rebindings := null;
-      Metadata      : Node_Metadata := Empty_Metadata;
-      Categories    : Ref_Categories;
-      Local_Results : in out Lookup_Result_Vector;
-      Toplevel      : Boolean := True)
+     (Self                    : Lexical_Env;
+      Key                     : Symbol_Type;
+      Lookup_Kind             : Lookup_Kind_Type := Recursive;
+      Rebindings              : Env_Rebindings := null;
+      Metadata                : Node_Metadata := Empty_Metadata;
+      Categories              : Ref_Categories;
+      Local_Results           : in out Lookup_Result_Vector)
+   is
+      Dummy : Boolean;
```

Comment:
Given that this boolean can be read but not written fisrt in Get_Internal_Impl, I suggest to initialize it to False here